docs/Commiting.md

```diff
@@ -0,0 +1,30 @@
+# Commits
+
+This project follows the [conventional commits](https://www.conventionalcommits.org) guidelines to automate our version management and changelog generation.
+A pre-commit hook runs [commit lint](https://commitlint.js.org/) before every commit to ensure commit consistency.
+If you need some help commiting, try running `yarn commit` to commit using the [commitizen](https://github.com/commitizen/cz-cli) tool.
+
+> Be sure to check out those links for more information on how conventional commits guidelines and tools
+
+## Commit requirements
+
+- A commit message should follow the conventional commits guidelines
+- A commit message should include an issue number (DSY-*)
```

Comment:
We should also let people reference a issue on github issues.
If this is too much of a work... lets add a warning note, something like "Not yet ready for external contributions. If you'd like to contribute and you are not a developer on Natura organization, please open a issue for us. We'd like to discuss our contribution proccess"

Reply:
Adicionei a nota, done in c5e361b